app/src/main/java/com/github/kulebin/myfoursquareapp/VenueDetailFragment.java

```diff
@@ -53,6 +54,8 @@ public void onViewCreated(final View view, @Nullable final Bundle savedInstanceS
 
         if (getArguments() != null) {
             mPresenter.onViewCreated(getArguments().getString(VENUE_ID));
+        } else {
+            Toast.makeText(getContext(), R.string.ERROR_TEXT_NO_VENUE_ID, Toast.LENGTH_LONG).show();
         }
```

Comment:
//TODO  make common error to show when venueId doesn't exist or corrupted when implement error handling